module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -735,7 +766,7 @@ protected function formatHoldingItem(
             'status' => $item->status->name,
             'duedate' => $dueDateValue,
             'availability' => $item->status->name == 'Available',
-            'is_holdable' => $this->isHoldable($locationName),
+            'is_holdable' => $this->isHoldable($locationName, $currentLoan),
```

Comment:
I considered doing the AND conditional here, and having `isHoldable` be a deprecated method that just calls `isHoldableLocation` for back compatibility until we remove it.  But I figured it's better to keep the complexity inside a single function.

Reply:
Yes, I think that's reasonable, especially because you've kept the interface of isHoldable back-compatible. The behavior only changes if you pass the extra argument.